pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -518,6 +577,45 @@ func (a *cpuAccumulator) takeFullSockets() {
 		a.take(cpusInSocket)
 	}
 }
+func (a *cpuAccumulator) takeFullUnCore() {
+	for _, uncore := range a.freeUnCoreCache() {
+		cpusInUnCore := a.topo.CPUDetails.CPUsInUnCoreCaches(uncore)
+		if !a.needsAtLeast(cpusInUnCore.Size()) {
+			continue
+		}
+		klog.V(4).InfoS("takeFullCCD: claiming CCD", "uncore", uncore)
```

Comment:
let's add consistent logging to what other functions have, and IIRC we don't add logging here

Reply:
Is there specific docs that talk about where we are suppose to log? I'm seeing it happen in the other similar spots this looks consistent

Reply:
ok, let's postpone this for now